sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -119,6 +119,14 @@ page at http://checkstyle.sourceforge.net/config.html -->
       <property name="message" value="You are using raw guava, please use vendored guava classes."/>
     </module>
 
+    <!-- Forbid use of Charset.defaultCharset()-->
+    <module name="RegexpSinglelineJava">
+      <property name="id" value="ForbidCharset"/>
```

Comment:
nit: ultra minor (you can ignore me but good if changed) s/ForbidCharset/ForbidDefaultCharset

Reply:
Updated, thanks!